index.test-d.ts

```diff
@@ -9,6 +9,7 @@ import {
 const error = new Error('unicorn');
 
 expectType<number>(serializeError(1));
+expectType<unknown>(serializeError(error as unknown));
 expectType<ErrorObject>(serializeError(error));
```

Comment:
This isn't enough actually. `expectType<unknown>` works for all three expectations, and so does `expectAssignable<unknown>`

What's the right assertion?

Reply:
- https://github.com/tsdjs/tsd/issues/212

Reply:
Try https://github.com/mmkal/expect-type instead. It has better assertions.

Reply:
Indeed